tests/App/Models/GalleryTest.php

```diff
@@ -382,6 +382,8 @@
 ]);
 
 it('determines if a gallery was reported recently by the user', function ($hoursAgo, $expected) {
+    $reportTimestamp = now()->subHours($hoursAgo);
```

Comment:
how does moving it here fix the issue? 🤔 the test itself doesn't reuse the timestamp so I don't follow how moving it up would make it less flaky

Reply:
won't moving it up make it be more affected by the user and gallery creation? start of the method it's 48 hours from now, then user + gallery gets created and it's 48 hours + time it took for that to be done. 

Maybe you can have a look whether `Carbon::setTestNow()` would help with this to set it to a specific value to test against, otherwise go with your suggestion of changing the intervals slightly to have more time to run the test in itself

Reply:
further context, test still fails for me on this branch:
```
Tests\App\Models\GalleryTest > it determines if a gallery was reported recently by the user (48, true) @ repetition 22 of 100
  Failed asserting that false is identical to true.
  ```